.browserslistrc

```diff
@@ -1,6 +1,12 @@
-last 2 versions
+
```

Comment:
I don't think a new line is necessary at the top of this file :)

Reply:
Thanks!